htdocs/class/logger/filelogger.php

```
@@ -0,0 +1,583 @@
+            $this->writeFailed = true;
+
+            return;
+        }
+        // Non-blocking: a lock held by a stalled process must not park this request in a
+        // queue behind it. Losing one debug line beats holding a worker.
+        if (flock($handle, LOCK_EX | LOCK_NB)) {
+            fwrite($handle, $entry);
+            flock($handle, LOCK_UN);
+        }
+        fclose($handle);
+    }
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 513. (undefined)

(ErrorControlOperator)